src/WebJobs.Script/Utility.cs

```diff
@@ -879,6 +881,31 @@ public static bool IsResourceAzureBlobWithoutSas(Uri resourceUri)
             return true;
         }
 
+        public static string GetSasTokenExpirationDate(Uri resourceUri)
+        {
+            // Screen out URLs with an SAS token
+            var queryParams = HttpUtility.ParseQueryString(resourceUri.Query.ToLower());
+            if (queryParams != null && !string.IsNullOrEmpty(queryParams[SasTokenExpirationDate]))
+            {
+                return queryParams[SasTokenExpirationDate];
+            }
+
+            return string.Empty;
+        }
+
+        public static string GetSasTokenExpirationDateFromSasSignature(string azureWebJobsStorage)
+        {
+            var splitStorage = azureWebJobsStorage.Split(';');
+            var sasToken = splitStorage[splitStorage.Length - 1];
```

Comment:
This method assumes that the "SharedAccessSignature" segment is the last item in the `;` separated string. Is it possible to get a string with different order or segments (SharedAccessSignature segment is before TableEndpoint segment) ?

Reply:
Discussed offline and the format of this string will always be the same according to this documentation: https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#create-a-connection-string-using-a-shared-access-signature